zkevm-circuits/src/evm_circuit/util/math_gadget.rs

```diff
@@ -26,7 +27,10 @@ mod test_util;
 
 pub(crate) use abs_word::AbsWordGadget;
 pub(crate) use add_words::AddWordsGadget;
-pub use batched_is_zero::BatchedIsZeroGadget;
+#[allow(unused_imports)]
+pub(crate) use batched_is_zero::BatchedIsZeroGadget;
+#[allow(unused_imports)]
+pub(crate) use binary_number::BinaryNumberGadget;
```

Comment:
Why do we need this if is not used? Can't we add it under the exact feature that requires them or just remove them??
We can export them when needed.